### PR TITLE
whitelist bitsandbytes

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist bitsandbytes

### Linked issues
Progresses #1901 

### Functionality 
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known-readme.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Known libraries that are not in known.json
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known-readme.txt:1]: remove this file

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/source_code/known-readme.txt (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known-readme.txt:1]: done

**Review by nfx — APPROVED** [review]
lgtm

**Comment by nfx** [comment]
make CI pass

**Comment by ericvergnaud** [comment]
> make CI pass

done

**Pull request merged**